src/Symfony/Bridge/Doctrine/Id/UuidGenerator.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Types;
+
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\Id\AbstractIdGenerator;
+use Symfony\Component\Uid\UuidV4;
+
+final class UuidGenerator extends AbstractIdGenerator
```

Comment:
shouldn't we have `UuidV4Generator` but also `UuidV1Generator` and `UuidV6Generator`? (no V3 nor V5)